app/uk/gov/hmrc/helptosavestub/controllers/NSIGetAccountBehaviour.scala

```diff
@@ -254,9 +247,52 @@ object NSIGetAccountBehaviour {
                                                                Term(2, LocalDate.of(2016, 3, 1), LocalDate.of(2916, 2, 28), "2400.00", "0.00", "0.00"))
 
     def zeroBonusPositiveBalanceResponse(correlationId: Option[String]): NSIGetAccountByNinoResponse = NSIGetAccountByNinoResponse("V1.0", correlationId,
-      "1100000112062", "2400.00", "2400.00", " ", None, None, "00", "00", zeroBonusPositiveBalanceCIM, "FirstTerm", "Saver", LocalDate.of(1963, 11, 1), "Line 1", "Line 2",
+      "1100000112070", "2400.00", "2400.00", " ", None, None, "00", "00", zeroBonusPositiveBalanceCIM, "FirstTerm", "Saver", LocalDate.of(1963, 11, 1), "Line 1", "Line 2",
       " ", " ", " ", "SV1 1QA", "GB", Some("email.address@domain.com"),
       "02", "00", "00", " ", "11111111", "Mr P Smith", None, "801497", zeroBonusPositiveBalanceTerms)
 
+    val closedCIM: CurrentInvestmentMonth = CurrentInvestmentMonth("0.00", "50.00", LocalDate.of(2018, 3, 31))
+
+    val closedTerms: List[Term] = List[Term](Term(1, LocalDate.of(2017, 11, 1), LocalDate.of(2019, 10, 31), "250.00", "125.00", "0.00"),
```

Comment:
Looks like the existing "closed account" have `bonusEstimate` value incorrect. 

Instead of "125.00", it should be "0.00" so the behaviour is consistent with all the new "closed account"

Reply:
I have updated to 0.00